Lib/tkinter/test/test_tkinter/test_misc.py

```diff
@@ -201,6 +201,14 @@ def test_clipboard_astral(self):
             root.clipboard_get()
 
     def test_winfo_rgb(self):
+
+        def assertApprox(col1, col2):
+            # A small amount of flexibility is required (bpo-45496)
+            # 33 is ~0.05% of 65535, which is a reasonable margin
+            self.assertTrue(all(
+                abs(i - j) < 33 for i, j in zip(col1, col2)
+            ))
```

Comment:
A report in case of failure will be not informative ("False is not True" or like). I suggest to use three separate assertAlmostEqual() for different components.

Reply:
Changed to use a `for` loop and `assertAlmostEqual`